cg/constants/taxprofiler.py

```diff
@@ -0,0 +1,22 @@
+"""Taxprofiler related constants."""
+
+
+from cg.constants.nextflow import NFX_SAMPLE_HEADER, NFX_SAMPLESHEET_READS_HEADERS
+
+TAXPROFILER_RUN_ACCESSION = "run_accession"
+TAXPROFILER_INSTRUMENT_PLATFORM = "instrument_platform"
+TAXPROFILER_ACCEPTED_PLATFORMS = ["ILLUMINA", "OXFORD_NANOPORE"]
+TAXPROFILER_FASTA_HEADER = "fasta"
+TAXPROFILER_SAMPLESHEET_HEADERS = [
+    NFX_SAMPLE_HEADER,
+    TAXPROFILER_RUN_ACCESSION,
+    TAXPROFILER_INSTRUMENT_PLATFORM,
+]
+TAXPROFILER_SAMPLESHEET_HEADERS.extend(NFX_SAMPLESHEET_READS_HEADERS)
+TAXPROFILER_SAMPLESHEET_HEADERS.append(TAXPROFILER_FASTA_HEADER)
+
+
+class TaxprofilerDefaults:
+    """Taxprofiler default parameters"""
+
+    INSTRUMENT_PLATFORM: str = "ILLUMINA"
```

Comment:
Your solution of including it in the common constants is better! You are right that it's not specific to taxprofiler.

But then I think you can remove completly TaxprofilerDefaults, since it is not used anywhere

Reply:
Yes, I think I can remove TaxprofilerDefaults.